tests/test_official.py

```diff
@@ -107,6 +107,8 @@ def check_object(h4):
             field = 'from_user'
         elif name.startswith('InlineQueryResult') and field == 'type':
             continue
+        elif name.startswith('InputMedia') and field == 'type':
```

Comment:
Could you combine with above? 
```elif (name.startswith('InlineQueryResult') or name.startswith('InputMedia')) and field == 'type':```
seems cleaner to me.

Reply:
yes, I pondered on it, but I will